src/components/ProfileContent/ProfileContent.tsx

```diff
@@ -0,0 +1,241 @@
+                reference={firstNameRef}
+                placeholder={user.firstName || ''}
+                defaultValue={user.firstName || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Country</Col>
+            <Col>
+              {' '}
+              <InputField
+                className={`profile-form ${
+                  !user.country && 'profile-form--empty'
+                }`}
+                reference={countryRef}
+                placeholder={user.country || 'your country'}
+                defaultValue={user.country || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Job Role</Col>
+            <Col>
+              {' '}
+              <InputField
+                className={`profile-form ${
+                  !user.jobRole && 'profile-form--empty'
+                }`}
+                reference={jobRoleRef}
+                placeholder={user.jobRole || 'your job role'}
+                defaultValue={user.jobRole || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+        </Col>
+        <Col sm>
+          <Row>
+            <Col>Last Name</Col>
+            <Col>
+              {' '}
+              <InputField
+                className="profile-form"
+                reference={lastNameRef}
+                placeholder={user.lastName || ''}
+                defaultValue={user.lastName || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Birthday</Col>
+            <Col>
+              {' '}
+              <InputField
+                className={`profile-form ${
+                  !user.birthday && 'profile-form--empty'
+                }`}
+                reference={birthdayRef}
+                placeholder={user.birthday || 'your birthday'}
+                defaultValue={user.birthday || ''}
+                readOnlyValue={props.isProfileInEditMode}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>Email</Col>
+            <Col className="first-row__email-column">{user.email}</Col>
+          </Row>
+        </Col>
+        {!props.isProfileInEditMode && (
+          <Row className="first-row__buttons">
+            {errorMsg && (
+              <div className="first-row__buttons--error error-message">
+                {errorMsg}
+              </div>
+            )}
+
+            <Button
+              className="btn-secondary mt-2 first-row__buttons--btn btn-cancel"
+              onClick={cancelEditProfile}
+            >
+              Cancel
+            </Button>
+            <Button
+              className="btn-primary mt-2 first-row__buttons--btn"
+              onClick={updateProfileFieldsDb}
```

Comment:
On click Save we should show a loader for user to know that something is happening and hide the functionality to change the fields.